src/registry.rs

```diff
@@ -786,395 +859,6 @@ pub enum MockReference {
     Attribute,
```

Comment:
I think `MockReference`, `Branch`, etc. (the enums used as members on the `Violation` structs) should be moved out of this file.

Reply:
I agree but I'd like to leave moving stuff around between modules to followup PRs.

Reply:
Ok, no prob.